### DISCUSSION TITLE
Nomination of @wborn

### DISCUSSION THREAD

**Discussion by kaikreuzer** [openhab/openhab-core#3493] [Core Maintainers Discussions]
As mentioned [here](https://github.com/orgs/openhab/teams/core-maintainers/discussions/1), I would like to increase the maintainer team of openhab-core and nominate @wborn  to be a maintainer of openhab-core. Wouter did an [incredible amount of contributions to ESH](https://github.com/eclipse/smarthome/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aclosed+author%3Awborn) in the past and he always proved deep understanding of the code and did very diligent changes.

**Comment by kaikreuzer** [comment]
Welcome, @wborn, I have added you to the team!

**Comment by wborn** [comment]
Thanks for the confidence and adding me to the team!
